core/embed/rust/src/ui/model_tt/component/button.rs

```diff
@@ -252,7 +259,9 @@ impl Component for Button {
                         // Touch started in our area, transform to `Pressed` state.
                         if touch_area.contains(pos) {
                             #[cfg(feature = "haptic")]
-                            play(HapticEffect::ButtonPress);
+                            if self.haptics {
+                                play(HapticEffect::ButtonPress);
```

Comment:
I would prefer to import the `haptic` module instead of using the `play` function directly. Then, we could call `haptic::play()` and eventually `haptic::play_rtp()`, which I find clearer. This can be applied to `loader.rs` as well - where these two approaches are combined now.

Reply:
fixed https://github.com/trezor/trezor-firmware/commit/f42d8f7ce5ec838d8b5021d26404dddb1a907610 and https://github.com/trezor/trezor-firmware/commit/9e492945a6e334827703b740b36876f8ef514ec1